servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered_inc.glsl

```diff
@@ -96,7 +96,7 @@ struct Lightmap {
 	mat3 normal_xform;
 	vec2 light_texture_size;
 	float exposure_normalization;
```

Comment:
It might be a good idea to add a blank line between each block of 16 bytes as follows:

```suggestion
	mat3 normal_xform;

	vec2 light_texture_size;
	float exposure_normalization;
```